components/camera/videosource/webcam.go

```diff
@@ -76,6 +77,11 @@ type CameraConfig struct {
 	Properties []prop.Media
 }
 
+func getLabelFromDriverInfo(info driver.Info) string {
```

Comment:
why the last?

Reply:
Was mostly trying to be consistent with `Discover` below. 

Though, it has the side effect of preferring a devices `/dev/video*` path over its USB port. Using any part should work and I'm realizing we should prefer the USB port over the more ephemeral `/dev/video*` part after discussions with CV Friday. Will change to the grab the first part. It makes no difference for Darwin and Windows since both OSes only have one part to their label.